### ISSUE TITLE
JavaScript formatting error

### ISSUE THREAD

**Issue by egamma** [Microsoft/TypeScript#7962]
_From @aayushkapoor206 on April 7, 2016 19:33_

- VSCode Version: 0.10.11
- OS Version: Windows 10

Steps to Reproduce:

1. Before
![before](https://cloud.githubusercontent.com/assets/13454335/14364253/a53ee48c-fd25-11e5-856a-c556e400f1f0.png)

2. After
![after](https://cloud.githubusercontent.com/assets/13454335/14364269/adbddb7c-fd25-11e5-8acd-2478c9ee2dfe.png)
 

_Copied from original issue: Microsoft/vscode#5086_

**Comment by egamma** [comment]
_From @mbana on April 7, 2016 20:6_

Just adding to this request...

I've tried to diagnose my problem but I can't step through the (minified) VSCode JS code to see what's going on. My code is just refusing to format. Period. The only thing I get in the developer tools:

``WARN: Ignoring commands``

Could you please make a debug version of VSCode avaiable for download on Linux? I believe this might be better in the long run as for the project. I'd be more than happy to look through the problem, seeing as I use the IDE a lot, once this is ready.

Many thanks

**Comment by egamma** [comment]
The JavaScript formatter is contributed by TypeScript, moving to the TypeScript project for investigation.

@mbana did you try to debug the typescript server or the Code core/renderer process (https://github.com/Microsoft/vscode/wiki/How-to-Contribute#debugging)? The renderer and extension host process supports source maps

**Comment by SaschaNaz** [comment]
Duplicate of #6320?

**Comment by mhegazy** [comment]
in deed. closing in favor of #6320
